src/.editorconfig

```diff
@@ -231,7 +231,7 @@ csharp_style_expression_bodied_local_functions = false:silent
 csharp_style_throw_expression = true:suggestion
 csharp_indent_labels = no_change
 csharp_space_around_binary_operators = before_and_after
-csharp_indent_case_contents = false
+csharp_indent_case_contents = true
```

Comment:
>true [](http://example.com/codeflow?start=30&length=4)

your change is ok is it doesn't indent case contents that are braced. can we apply braces to the modified code and check that?